imap_processing/codice/l0/decom_codice.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+from imap_processing import imap_module_directory
+from imap_processing.decom import decom_packets
+
+if __name__ == "__main__":
+    # Define paths
+    packet_file = Path("housekeeping_data.bin")
+    xtce_document = Path(
+        f"{imap_module_directory}/codice/packet_definitions/P_COD_NHK.xml"
```

Comment:
Does this decom work on the new files in the `data/` directory? If so, maybe we should use those since they come from the 'official' CoDICE testing data